app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -58,6 +60,26 @@ def get_vms
     process_collection(@inv[:vms], :vms) { |vm| parse_vm(vm) }
   end
 
+  def get_private_images
+    @inv[:orgs].each do |org|
+      org.catalogs.each do |catalog|
+        get_images(catalog.catalog_items) unless catalog.is_published
+      end
+    end
+  end
+
+  def get_public_images
```

Comment:
These look like they could be one method and just `org.catalogs.select { |c| c.is_published == arg }`